spec/controllers/videos_controller_spec.rb

```diff
@@ -2,41 +2,37 @@
 
 describe VideosController do
   describe "GET show" do
+    let(:video) { Fabricate(:video) }
+    
     it "sets the @video for authenticated user" do
-      session[:user_id] = Fabricate(:user).id
-      video = Fabricate(:video)
+      set_current_user
```

Comment:
I would probably consider extracting this to a before block. You are using it almost everywhere.